ambari-server/src/main/resources/stack-hooks/before-ANY/scripts/params.py

```diff
@@ -242,8 +242,8 @@ def is_secure_port(port):
 
 user_to_gid_dict = collections.defaultdict(lambda:user_group)
 
-user_list = json.loads(config['hostLevelParams']['user_list'])
-group_list = json.loads(config['hostLevelParams']['group_list'])
+user_list = set(json.loads(config['hostLevelParams']['user_list']) + [smoke_user])
```

Comment:
Why is this change included in this PR?  It doesn't appear to be related to the other Blueprint-specific changes in this PR.

Reply:
At some point I ran into an error where `smoke_user` was not being created, but I think the role-command order workaround addresses it, too.  So we don't need this anymore.